l10n_it_withholding_tax/wizards/account_payment_register.py

```diff
@@ -0,0 +1,37 @@
+from odoo import api, fields, models
+
+
+class AccountPaymentRegister(models.TransientModel):
+    _inherit = "account.payment.register"
+
+    @api.depends(
+        "source_amount",
+        "source_amount_currency",
+        "source_currency_id",
+        "company_id",
+        "currency_id",
+        "payment_date",
+        "journal_id",
+    )
+    def _compute_amount(self):
```

Comment:
Come mai non viene chiamato `super`?

Reply:
La `super()` non viene chiamata, perché imposta il valore di `wizard.amount` a quello iniziale; il comportamento che si vuole ottenere è quello di mantenere l'importo messo a mano nel wizard quando si va a cambiare il `journal_id`. 
La modifica è sviluppata affinché se la fattura ha le RdA e viene registrato il pagamento, questo è al netto della ritenuta; mentre con la compute standard ciò non avviene

Reply:
Conviene comunque chiamare `super` prima di fare gli interventi, eventualmente salvando in una variabile i valori che serve ripristinare, perchè, se ci fosse qualche altro modulo che fa override per altri scopi, queste modifiche gli impedirebbero di funzionare.